### PR TITLE
Add Russian

### PR DESCRIPTION
**Pull request by ShevT**
By @ShevT

### CODE REVIEW

**Comment by adil192** [comment]
Awesome thank you!

File: lib/i18n/strings_ru.i18n.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
+{
+	"home": {
+		"tabs": {
+			"home": "Домашняя",
+			"browse": "Обзор",
+			"whiteboard": "Чистый лист",
```
Comment (adil192) [inline lib/i18n/strings_ru.i18n.json:6]: With Google Translate, "Чистый лист" translates to "a blank sheet (of paper)" whereas "whiteboard" translates to "белая доска". Is "Чистый лист" the better translation?

Comment (ShevT) [reply lib/i18n/strings_ru.i18n.json:6]: I translated the way it sounds right in Russian.

Comment (ShevT) [reply lib/i18n/strings_ru.i18n.json:6]: Do not fully trust the Google translator

Comment (adil192) [reply lib/i18n/strings_ru.i18n.json:6]: Thanks, no offense intended

**Comment by adil192** [comment]
Thanks for your efforts; I hadn't realised how big the language files had become!

Also would you be interested in being mentioned when there's new strings that need to be translated? https://github.com/adil192/saber/issues/19#issuecomment-1306096416

**Comment by ShevT** [comment]
> 
> Also would you be interested in being mentioned when there's new strings that need to be translated? [#19 (comment)](https://github.com/adil192/saber/issues/19#issuecomment-1306096416)

Yes, sure!

**Pull request merged**